src/db/file_transfer.py

```diff
@@ -1,6 +1,7 @@
 import paramiko
 import os
 import sys
+from src.db.config import get_config
 from contextlib import contextmanager
 
 from src.db.config import get_config
```

Comment:
`from src.db.config import get_config` appears twice

Reply:
Whoops, I'll fix this